src/imitation/scripts/common/reward.py

```diff
@@ -62,16 +72,43 @@ def config_hook(config, command_name, logger):
         res["net_cls"] = default_net
     if "normalize_input_layer" not in config["reward"]["net_kwargs"]:
         res["net_kwargs"] = {"normalize_input_layer": networks.RunningNorm}
+
+    if config["reward"]["net_cls"] is reward_nets.RewardEnsemble:
```

Comment:
This will only apply to `RewardEnsemble` itself, not subclasses -- is that desired?